# path_saver & path_recovery
update_min_d: 0.2
update_min_a: 0.7854

# path_load_segments
max_plan_length: 3
update_min_dist: 0.2
look_ahead_dist: 1.5
look_ahead_angle: 0.3
update_time_interval: 1.0

# path_recovery
check_tolerance: 0.4
skip_on_obstruction: false
obstruction_threshold: 70
clearing_timeout: 15.0
